orttraining/orttraining/test/training_api/common/synthetic_data_loader.h

```diff
@@ -15,31 +15,23 @@
 
 #include <memory>
 #include <utility>
+#include <variant>
```

Comment:
><variant> [](http://example.com/codeflow?start=9&length=9)

I can not find variant usage in this file. Perhaps, you need to included where it is needed?